src/routes/landing/components/CityEntry.js

```diff
@@ -31,11 +34,31 @@ type PropsType = {|
 class CityEntry extends React.PureComponent<PropsType> {
   render () {
     const { city, language, filterText } = this.props
+    const getMunicipality = (city, filterText): Array<CityModel> => {
+      if (!filterText) {
```

Comment:
You could check here for both `filterText` and `city._aliases`, so that only one if is needed.